katello/hooks/post/31-cdn_setting.rb

```diff
@@ -0,0 +1,9 @@
+if module_enabled?('katello')
+  param = get_custom_config('cdn_ssl_version')
+  if param
+    logger.info 'cdn_ssl_version param found, migrating to a Katello setting'
+    `foreman-rake -- config -k cdn_ssl_version -v #{param}`
```

Comment:
I'd lean towards calling our builtin `execute` so that this gets logged and fails if this command errors out.